tools/travis.sh

```diff
@@ -1,7 +1,10 @@
 set -x
 ulimit -n 4096
+# do this for our daily build test run
+if [[ "$TRAVIS_EVENT_TYPE" == "cron" ]]; then
+    powershell -c "Import-Module ./build.psm1; Start-PSBootstrap; Start-PSBuild -CrossGen; Start-PSPester -Tag @('CI','Feature','Scenario') -ExcludeTag RequireAdminOnWindows; Start-PSxUnit"
```

Comment:
I was wondering about that, my thought that was if we're not doing it for pull requests, i didn't want to do it here. I'm open to an alternative position, though. @andschwa do you have an opinion?